docker_explorer/lib/container.py

```diff
@@ -204,3 +215,41 @@ def GetHistory(self, show_empty_layers=False):
         else:
           history_str += 'Empty layer'
     return history_str
+
+  def _SetStorage(self, storage_name):
+    """Sets the storage_object attribute.
+
+    Args:
+      storage_name (str): the name of the storage.
+    Returns:
+      BaseStorage: a storage object.
+    Raises:
+      BadContainerException: if no storage Driver is defined, or if it is not
+        implemented
+    """
+    storage_class = self.STORAGES_MAP.get(storage_name, None)
+
+    if storage_class is None:
```

Comment:
You can also do `if storage_name not in self.STORAGES_MAP`, and then assume that the key `storage_name` will always be valid (e.g. access it using `self.STORAGES_MAP[storage_name]`)

Reply:
I'll keep it as is, which looks better than:
`self.obj = self.STORAGES_MAP[storage_name](arg1, arg2)`

Reply:
SG